app/collector.go

```diff
@@ -110,15 +110,11 @@ func (c *collector) Report(_ context.Context) (report.Report, error) {
 	}
 	c.reports = clean(c.reports, c.window)
 
-	rpt := report.MakeReport()
-	id := murmur3.New64()
+	rpts := []report.Report{}
 	for _, tr := range c.reports {
-		rpt = rpt.Merge(tr.report)
-		id.Write([]byte(tr.report.ID))
+		rpts = append(rpts, tr.report)
 	}
```

Comment:
There will be 15/3 * #hosts.  So could be lots.  I doubt the copy is expensive, but we might as well do this.